Mathlib/AlgebraicGeometry/Properties.lean

```diff
@@ -89,14 +89,14 @@ theorem isReducedOfOpenImmersion {X Y : Scheme} (f : X ⟶ Y) [H : IsOpenImmersi
       Y.presheaf.obj _ ≅ _).symm.commRingCatIsoToRingEquiv.injective
 #align algebraic_geometry.is_reduced_of_open_immersion AlgebraicGeometry.isReducedOfOpenImmersion
 
-set_option maxHeartbeats 250000 in
+set_option maxHeartbeats 300000 in
 instance {R : CommRingCat} [H : _root_.IsReduced R] : IsReduced (Scheme.Spec.obj <| op R) := by
   apply (config := { allowSynthFailures := true }) isReducedOfStalkIsReduced
   intro x; dsimp
-  have : _root_.IsReduced (CommRingCat.of <| Localization.AtPrime (PrimeSpectrum.asIdeal x)) := by
-    dsimp; infer_instance
-  exact isReduced_of_injective (StructureSheaf.stalkIso R x).hom
-    (StructureSheaf.stalkIso R x).commRingCatIsoToRingEquiv.injective
+  apply @isReduced_of_injective _ _ _ _ _ _ (StructureSheaf.stalkIso R x).hom
+    (StructureSheaf.stalkIso R x).commRingCatIsoToRingEquiv.injective ?_
+  dsimp
+  infer_instance
```

Comment:
Okay, just noting for myself: this change results in a slow-down on master, but a speed-up here.